src/routes/+page.svelte

```diff
@@ -1,3 +1,7 @@
+<script lang="ts">
```

Comment:
TypeScript isn't actually necessary here since we don't use any of its fancy syntax.
```suggestion
<script>
```